components/style/gecko/media_queries.rs

```diff
@@ -488,13 +570,17 @@ impl Expression {
     /// ```
     pub fn parse<'i, 't>(context: &ParserContext, input: &mut Parser<'i, 't>)
                          -> Result<Self, ParseError<'i>> {
-        input.expect_parenthesis_block()?;
+        input.expect_parenthesis_block().map_err(|_|
```

Comment:
Gecko does not report an error here, so we can revert this change.

Reply:
If I revert this, `@media "foo" {}` would only report "Declaration dropped" instead of "Expected identifier in media list but found ‘ "foo" ’"